lib/lighthouse.js

```diff
@@ -56,15 +61,25 @@ function run() {
 
   return executeSequentially(tasks)
     .then(() => {
-      const output = reportBuilder.build();
-      fs.writeFileSync(__dirname + '/../lighthouse-report.xml', output);
-      logger.log(output);
+      const outputJson = JSON.stringify(jsonOutput);
```

Comment:
Given this JSON output is an addition or "nice to have", maybe it would be good to make a flag to enable this so that we can save a bit of processing time for people that don't need this extra output? Perhaps not setting the `ATW_OUTPUT_JSON` variable could turn this functionality off?

Reply:
If the variable isn't set, currently the JSON will be saved [somewhere in `a11y-tests-web`](https://github.com/bbc/a11y-tests-web/pull/19/files#diff-e7b4ba32d74c8c13cb5f81f7a4f26e17R139) (we will set the path to be our executing repo). Are you proposing removing that default location?

Reply:
Yeah, I'm proposing making it so that if you don't set where you want it, it doesn't output it at all, as it's an additional feature. What do you think?

Reply:
@radiocontrolled what do you think?

Reply:
I like not getting JSON output unless a path is specified. We'd need to adapt the readme to explain it as an optional feature.